tests/listediting.js

```diff
@@ -325,8 +325,8 @@ describe( 'ListEditing', () => {
 
 	describe( 'flat lists', () => {
 		describe( 'setting data', () => {
-			function test( testName, string, expectedString = null ) {
-				it( testName, () => {
+			function test( testName, string, expectedString = null, skip = false ) {
```

Comment:
Instead of this you could just comment out a test or wrap it in `describe.skip()`.

Reply:
BTW. it reminds that something similar was somewhere... probably undo tests in lists were skipped this way... But there at least half of a test was performed.